src/components/Modal/Modal.component.js

```diff
@@ -14,7 +14,7 @@ const Modal = ({ isModalOpen, closeModal, children, className }) => {
         <Dialog.Overlay className="fixed inset-0 bg-black opacity-50" />
 
         <div
-          className={`relative bg-inverted rounded-xl p-6  max-w-sm mx-auto`}
+          className={`relative bg-[#3c3c3c] text-white rounded-xl p-6 max-w-sm mx-auto`}
```

Comment:
```suggestion
          className={`relative bg-hover text-white rounded-xl p-6 max-w-sm mx-auto`}
```

Reply:
https://github.com/cinemataztic/cine-ui/blob/master/tailwind.config.js#L27- This shade is specified in the config file, we can use it.